elastic/kibana
# [Synthetics] Disable edit policy

## Summary
Fixes https://github.com/elastic/kibana/issues/137505

Shows deprecation notice when attempting to edit legacy integrations

[Elastic-Synthetics---Integrations---Elastic (1).webm](https://user-images.githubusercontent.com/11356435/228042467-9b9a593a-a713-4a38-aa6a-1c083e82f3c5.webm)

### Testing
1. Check out the 8.5 branch
2. Create a legacy Elastic Synthetics integration
3. Check out this branch
4. Navigate to the legacy Elastic Synthetics integration
5. Observe the deprecation notice
